app/assets/stylesheets/modules/sitter.scss

```diff
@@ -129,13 +154,26 @@
         font-weight: $sitter-headline-weight;
       }
 
+      #image-upload {
+
+        ul li .postfix {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 7